pkg/noun/manage.c

```diff
@@ -617,11 +617,19 @@ _find_home(void)
     nor_w = (low_p + ((1 << u3a_page) - 1)) >> u3a_page;
```

Comment:
One more thing, though it's not exactly particular to this PR. Do snapshots include the extra 2 zero words at the beginning of the memory arena (for alignment), or are they included in the snapshot? If they're not included, this isn't exactly correct, since nor_w is just hat_p in this case (and not the north road length).

Reply:
-- looks like they do include the leading zeroes? Just want to confirm...

Reply:
The pages are aligned in memory and on disk, so the initial padding words *are* included. `low_p` is `u3R->hat_p - 1;` (the home road is always north); `nor_w` is the number of pages in the north segment (rounded up).